Deck.py

```diff
@@ -0,0 +1,145 @@
+# Deck class
+from TarotCard import *
+import random
+
+
+class Deck:
+    def __init__(self):
+        self.deck = []
+        self.i = 0

```

Comment:
A feature added in a newer version of python allows you to set these defaults outside of `__init__()`, which makes things a little cleaner:

```suggestion
class Deck:
    deck = []
    i = 0
    def __init__(self):
```

You can move the below lists outside of init like this too.